### PR TITLE
PROD updated muffnn to 1.1.2

### PR DESCRIPTION
**Pull request by beckermr**

### CODE REVIEW

**Comment by beckermr** [comment]
ping @jmemich - I think @olatop might be waiting on this.

**Comment by jmemich** [comment]
LGTM

@stephen-hoover -- not sure if we want to update the civis-model Docker image as 1.0.x bugfix but we could probably pip install the latest `muffnn` version.

@olatop what's the timeline on your project/needs with regard to this fix?

**Review by jmemich — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ Version number changes (major.minor.micro) in this package denote the following:
 
 ## Unreleased
 
+## [3.0.1] - XXXX-XX-XX
```
Comment (stephen-hoover) [inline CHANGELOG.md]: Are you proposing an immediate release for this bugfix?

**Review by stephen-hoover — COMMENTED** [review]
This is to incorporate the multi-output bugfix in `muffnn`, correct?

**Comment by olatop** [comment]
Not urgent at all on my end. @ckreyenhagen this is the fix we were discussing earlier

File: CHANGELOG.md (hunk 1 above)
Comment (beckermr) [reply CHANGELOG.md]: Nope. We can leave it untagged.

**Comment by beckermr** [comment]
Yes @stephen-hoover this is the multi-output bug fix.

File: CHANGELOG.md (hunk 1 above)
Comment (stephen-hoover) [reply CHANGELOG.md]: If it's not for immediate release, then this PR shouldn't include any version numbers. But we might as well make a release. Please put today's date here.

Comment (beckermr) [reply CHANGELOG.md]: Sounds good.

**Comment by beckermr** [comment]
@stephen-hoover this is done - LGTM when you are happy!

**Review by stephen-hoover — APPROVED** [review]
LGTM!

**Pull request merged**